src/controllers/userController.js

```diff
@@ -67,6 +92,16 @@ const updateCurrentUser = async (req, res) => {
 };
 
 const updateUserPassword = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch the current requesting user by id, update their password and save it to the database.'
+     #swagger.responses[200] = {
+				description: 'User successfully fetched and password updated.',
+				schema: { $ref: '#/definitions/PasswordlessUser' }
```

Comment:
This endpoint just returns a message object (`res.status(StatusCodes.`_`OK`_`).json({ msg: 'Success! Password Updated.' });`

```suggestion
				type: 'object'
				properties: {
				  msg: { type: 'string' }
				}
```